lib/ex_unit/test/ex_unit/capture_log_test.exs

```diff
@@ -81,6 +81,17 @@ defmodule ExUnit.CaptureLogTest do
     end
   end
 
+  test "exit with noproc when the logger is down" do
+    Application.stop(:logger)
```

Comment:
This actually prints a message on the stdio. Is there any other way to test this? Or maybe a way to suppress the stdio? A `capture_io/1` does the job, but it feels a little bit strange to capture the io just for not printing it.

Reply:
We have https://github.com/kelvinst/elixir/blob/64090d0c8c63109c2009e473cb28b347a9659e85/lib/logger/lib/logger/app.ex#L61 that we might be able to use. I think we need to ensure `:logger` is always started no matter what (an exit signal would prevent it). So for something that doesn't automatically cleaned up we should use `setup` or `on_exit` to ensure logger is always started before each test or that we ensure its started after this test.

Reply:
Done!